libs/sdk-react-native/ios/BreezSDKMapper.swift

```diff
@@ -3018,6 +3023,15 @@ class BreezSDKMapper {
                 "type": "endpointError",
                 "data": dictionaryOf(lnUrlErrorData: data),
             ]
+
+        case let .payError(
+            paymentHashreason
```

Comment:
Typo?

Reply:
Seemed to be an issue with the rn codegen. I moved the `LnUrlPayResult` data into wrapper structs, which resolved the issue.

Reply:
I'll investigate the RN codegen issue

Reply:
I think your change is the preferred (common) way we handle interface variants anyway. But it seems variants with multiple tuple/struct like data attributes are not being rendered correctly in swift.